frontend/client/components/Proposal/CampaignBlock/index.tsx

```diff
@@ -168,7 +180,9 @@ export class ProposalCampaignBlock extends React.Component<Props, State> {
                 />
               </div>
 
-              <Form layout="vertical" className="ProposalCampaignBlock-contribute">
+              {/* TODO: use this as a base for tipjar? */}
```

Comment:
Left this in as a reminder that it might help when implementing the tip jar. Can be removed before merge if desired.

Reply:
#6